docs/_sidebar.md

```diff
@@ -34,6 +34,6 @@
 * Workflow
 	* [Lookup table](workflow/lookup-table.md)
 	* [Math methods](workflow/math.md)
-	* [Memoization](workflow/memoization.md)
+	* [Memorization](workflow/memorization.md)
```

Comment:
I think memoization is correct: https://www.google.es/search?q=memoization&oq=memoization&aqs=chrome..69i57j69i60l2j0l3.157j0j1&sourceid=chrome&ie=UTF-8